.gitlab/deploy_dca/container.yml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
This file is currently not being imported anywhere, therefore it won't be used.

As this file is simple, and I don't expect more jobs to be added to this stage, I recommend to move this file to `.gitlab/deploy_dca.yml` (no need for a more complex directory structure for stages with only a few jobs owned by a single team), and then add `.gitlab/deploy_dca.yml` to the list of files included by the main `.gitlab-ci.yml`: https://github.com/DataDog/datadog-agent/blob/2ddc6975332c54390a1bc99d14750daa41d1ae57/.gitlab-ci.yml#L2-L36